src/app/messages.ts

```diff
@@ -38,12 +60,39 @@ export const errMessages = {
     }
   },
   Login: {
-    
+    "login": "Oops... Invalid email or password, please type it again."
+  },
+  PasswordValidation: {
+    mismatch: {
+      "mismatch": "Password mismatch ..."
+    },
+    minlength: {
+      "minlength": "Password minimum length is 8 characters ..."
+    }
   },
   Registration: {
-
+    mismatch: {
+      "match": "The passwords entered do not match."
```

Comment:
The passwords you have entered do not match each other. Please enter the same password.